package.json

```diff
@@ -19,7 +19,9 @@
   },
   "devDependencies": {
     "code": "^1.5.0",
+    "eslint": "^1.8.0",
     "eslint-config-nodesecurity": "^1.1.0",
+    "eslint-plugin-hapi": "^2.0.0",
```

Comment:
I think ESLint tries to resolve modules in the root of node_modules and not nested within other dependencies (sadly). I don't think this worked unless I explicitly added the **eslint-plugin-hapi** as a top level dependency (but I can recheck).